pkg/cloudprovider/metal/instances_v2_test.go

```diff
@@ -207,7 +207,7 @@ var _ = Describe("InstancesV2", func() {
 				Name: "foo",
 			},
 			Spec: corev1.NodeSpec{
-				ProviderID: getProviderID(ns.Name, "bar"),
+				ProviderID: fmt.Sprintf("%s://%s/%s/%s", ProviderName, ns.Name, "foo", "bar"),
```

Comment:
Nitpick: `fmt.Sprintf("%s://%s/foo/bar", ProviderName, ns.Name)`

Reply:
No, that is actually is a good catch.

Reply:
Fixed that one.